source/expressions.tex

```diff
@@ -884,12 +884,10 @@
 
 \pnum
 \indextext{conversion!bool@\tcode{bool}}%
-If the source type is \tcode{bool}, the value \tcode{false} is converted to
-zero and the value \tcode{true} is converted to one.
-
-\pnum
 \indextext{conversion!to unsigned}%
 \indextext{conversion!to signed}%
+If the source type is \tcode{bool}, the value \tcode{false} is converted to
```

Comment:
I like having two separate paragraphs here; bool is indeed handled totally differently from any other integer.